pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -42,23 +42,25 @@ const (
 	trustedCABundleConfigMap = "config-trusted-cabundle"
 	clusterInterceptors      = "openshift-pipelines-clusterinterceptors"
 	namespaceVersionLabel    = "openshift-pipelines.tekton.dev/namespace-reconcile-version"
+	disableRbacInjection     = "openshift-pipelines.tekton.dev/disable-rbac-injection"
```

Comment:
usually, double negatives could be confusing eg: `someting-disable: false` 
ie, disableRbac could be read as "disable doing something to do with RBAC" or there is an already existing entity which is disabling RBAC and this flag disbleRBAC will disable that.

if we can make this a neutral staement which toggles between true and false then it will be easier for our end user to think about it.

eg: <funciton-of-feature>: true/false